### PR TITLE
documentation updates for Tanzu

### PR DESCRIPTION
**Pull request by antyona**
@adamdyszy @Lukasz-Gebarowski Can you please review the PR with the doc updates for Tanzu? 

Related to: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/54227

### CODE REVIEW

File: docs/Content/Configuration.md [hunk 1]
```diff
@@ -128,6 +128,38 @@ spec:
 EOF
 ```
 
+- VMware Tanzu Kubernetes Grid
+
+Before you retrieve the IP, check your current network configuration.
+
+To get the load balancer IP, and set is as a variable, run the following command:
```
Comment (adamdyszy) [inline docs/Content/Configuration.md:135]: set it

**Review by adamdyszy — CHANGES_REQUESTED** [review]
one typo, apart from that everything good

File: docs/Content/Configuration.md (hunk 1 above)
Comment (antyona) [reply docs/Content/Configuration.md]: Corrected 👍

**Pull request merged**